src/main/scala-2.11/markets/MarketLike.scala

```diff
@@ -0,0 +1,27 @@
+package markets
+
+import akka.actor.ActorRef
+
+import markets.clearing.engines.MatchingEngine
+import markets.orders.Order
+import markets.tradables.Tradable
+
+/** Mixin Trait providing MarketLike behavior to some BaseActor. */
+trait MarketLike {
+  this: BaseActor =>
+
+  def clearingMechanism: ActorRef
+
+  def tradable: Tradable
+
+  def marketActorBehavior: Receive = {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Default case not specified.](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1560525557/issues/source?bid=2921361&fileBranchId=3019109#l17)